### PR TITLE
add azerbaijani missing translation

### LINKED ISSUE #45125: Missing translations for Azerbaijani (az)
Hello,

There are some missing translation for Azerbaijani (az). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.az.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.az.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.az.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.az.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by lacatoire**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #45125
| License       | MIT

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
How do you do those translations @lacpandore? We require a review by a native speaker before merging translations. Can you please figure out who would be a native speaker of each language and ask them a review here? I suggest looking at the "blame" of the file to figure this out. Please do that for every language you're contributing.

File: src/Symfony/Component/Stopwatch/Tests/StopwatchPeriodTest.php [hunk 1]
```diff
@@ -63,6 +63,6 @@ public function provideDurationValues()
         yield [2, 3.14, true, 1.1400000000000001];
         yield [2, 3.13, true, 1.13];
         yield [2.71, 3.14, false, 1];
-        yield [2.71, 3.14, true, 0.43];
+        yield [2.71, 3.14, true, 0.43000000000000016];
```
Comment (stof) [inline src/Symfony/Component/Stopwatch/Tests/StopwatchPeriodTest.php]: this change does not belong to a PR updating translations

**Comment by lacatoire** [comment]
> How do you do those translations @lacpandore? We require a review by a native speaker before merging translations. Can you please figure out who would be a native speaker of each language and ask them a review here? I suggest looking at the "blame" of the file to figure this out. Please do that for every language you're contributing.

ok i"ll do this

@seferov would you accept to see this PR to review the translation ?

File: src/Symfony/Component/Security/Core/Resources/translations/security.az.xlf [hunk 2]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Yanlış və ya müddəti keçmiş giriş keçidi.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Həddindən artıq uğursuz giriş cəhdləri, %minutes% dəqiqə ərzində yenidən cəhd edin.</target>
```
Comment (seferov) [inline src/Symfony/Component/Security/Core/Resources/translations/security.az.xlf:75]:
```suggestion
                <target>Həddindən artıq uğursuz giriş cəhdi, lütfən %minutes% dəqiqə ərzində yenidən yoxlayın.</target>
```

File: src/Symfony/Component/Security/Core/Resources/translations/security.az.xlf [hunk 3]
```diff
@@ -70,6 +70,14 @@
...
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Həddindən artıq uğursuz giriş cəhdləri, %minutes% dəqiqə ərzində yenidən cəhd edin.</target>
```
Comment (seferov) [inline src/Symfony/Component/Security/Core/Resources/translations/security.az.xlf:79]:
```suggestion
                <target>Həddindən artıq uğursuz giriş cəhdi, lütfən %minutes% dəqiqə ərzində yenidən yoxlayın.</target>
```

**Review by seferov — CHANGES_REQUESTED** [review]

**Comment by seferov** [comment]
> @seferov would you accept to see this PR to review the translation ?

@lacpandore I corrected some minor mistakes but the rest looks good.

**Comment by fabpot** [comment]
@seferov Can you submit a new PR so that we can merge it? Thank you.

**Comment by seferov** [comment]
> @seferov Can you submit a new PR so that we can merge it? Thank you.

@fabpot here it is https://github.com/symfony/symfony/pull/47875

**Comment by fabpot** [comment]
Closing in favor of #47875

**Pull request closed without merging**